test/run.d

```diff
@@ -163,8 +163,14 @@ Options:
     auto targets = args
         .predefinedTargets // preprocess
         .array
-        .filterTargets(env)
-        .schwartzSort!(sortKey, "a < b", SwapStrategy.stable);
+        .filterTargets(env);
+
+    // Do a manual schwartzSort until all host compilers have a fix
+    // for the invalid "lhs internal pointer" error
```

Comment:
Maybe reference the PR and dmd version so that people know when it can be removed?

Reply:
Good idea, added a link + estimated version